apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerTestDatabase.cs

```diff
@@ -95,6 +98,11 @@ private SpannerTestDatabase(string projectId)
             ConnectionString = databaseBuilder.ConnectionString;
             DatabaseName = databaseBuilder.DatabaseName;
 
+            if (SpannerClientCreationOptions.UsesEmulator)
```

Comment:
I would move this `if` to inside the method and always call the method, which won't do anything if the emulator is not being used.

Reply:
Done